### DISCUSSION TITLE
Prefixes not working

### DISCUSSION THREAD

**Discussion by ShotySSH** [jagrosh/MusicBot#1232] [Troubleshooting] [answered]
First, I want to 10q for such a wonderful bot. But set prefixes in the config file do not work on any of my servers. Altprefixes too. Commands are executed exclusively through the mention of the bot. Aplying the rest of the config settings works fine.

UPD: Forgot to say that commands like "setprefix" work. I get a return message, and in the serversettings file I see the prefix set on, but it does not work

Thanks in advance

**Comment by MichailiK** [comment, marked as answer]
#1215

**Reply by ShotySSH** [reply]
Very thank u
